core/templates/dev/head/pages/exploration_editor/history_tab/HistoryTab.js

```diff
@@ -53,6 +53,9 @@ oppia.controller('HistoryTab', [
     var explorationSnapshots = null;
     var versionTreeParents = null;
     var nodesData = null;
+    $scope.versionNumbersToDisplay = [];
+    $scope.currentPage = 1;
```

Comment:
Would expect this to be zero-indexed in the code (just do the transformation to 1-indexed in the UI). If you want to keep it 1-indexed then do $scope.currentPage1Indexed or similar.

Reply:
Done.